app/src/components/v-select/select-list-item-group.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-	<v-list-group :clickable="item.selectable" :value="item.value">
+	<v-list-group :clickable="item.selectable" :open="item.children?.length === 1" :value="item.value">
```

Comment:
@azrikahar Can you explain why you added the `:open` prop?

I use `v-select` on my fork of Directus for a custom use and this line causes issues for me. I have some items with only 1 child and they can't be expanded/collapsed as a result. See the below screenshot.

![image](https://user-images.githubusercontent.com/1342819/176978602-f31cd834-b758-45c9-93e4-89ae0d8050fb.png)

Reply:
It was based on [the suggestion above](https://github.com/directus/directus/pull/13867#issuecomment-1153824404), and originally #13752 does make it possible to close it as the logic was within v-list-group! However later on it was moved to v-checkbox-tree, which then subsequently made this change not work as intended anymore as you pointed out 😕

Probably best to remove this line for now, in favor of a potential v-select/v-checkbox-tree refactoring discussed internally in the future.